two_factor/templates/two_factor/_wizard_actions.html

```diff
@@ -1,14 +1,21 @@
 {% load i18n %}
 
-<div class="button-row">
-  {% if cancel_url %}
-    <a href="{{ cancel_url }}" class="pull-right btn btn-link">{% trans "Cancel" %}</a>
-  {% endif %}
-  {% if wizard.steps.prev %}
-    <button name="wizard_goto_step" type="submit" value="{{ wizard.steps.prev }}" class="btn btn-default">{% trans "Back" %}</button>
-  {% else %}
-    <button disabled name="" type="button" class="btn btn-disabled">{% trans "Back" %}</button>
-  {% endif %}
-  {% comment %} <a href="{% url 'two_factor:setup' %}" class="btn btn-default">{% trans "Setup" %}</a> {% endcomment %}
-  <button type="submit" class="btn btn-primary">{% trans "Next" %}</button>
+{% if wizard.steps.current == 'auth' %}
+    <div class="password-reset-link">
+        <a href="/admin/password_reset/">Forgotten your password or username?</a>
+    </div>
+{% endif %}
+<div class="actions-buttons">
```

Comment:
I think this class can be replaces with django admin `.submit-row` class

Reply:
the class submit-row is great to center one button, but not a group of button together